handbook/handbook.in

```diff
@@ -309,7 +309,7 @@ the theme's directory to <em>/usr/local/share/slim/themes/</em>. To see a previe
 in <em>nomadbsd-dmconfig</em> copy a screenshot of the login screen to
 <em>/usr/local/share/slim/themes/your-theme-name/preview.png</em>.</p>
 
-<p><img src="images/nomadbsd-dmconfig-screenshot.png" alt="" />
+<p><img src="handbook/images/nomadbsd-dmconfig-screenshot.png" alt="" />
```

Comment:
Same here.

Reply:
Thanks for the great feedback! Fixed :-)